ostelco-ios-client/Network/Analytics.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  Analytics.swift
+//  ostelco-ios-client
+//
+//  Created by mac on 4/29/19.
+//  Copyright © 2019 mac. All rights reserved.
+//
+
+import Foundation
+import ostelco_core
+import FirebaseAnalytics
+
+class ROAnalytics {
```

Comment:
Firebase calls their analytics class for Analytics, so decided to prefix with our internal name RO (RedOtter)

Reply:
Yeah that drives me batty, they really need to prefix that shit with `FIR`. 

FWIW I've been prefixing with `Ostelco` for anything that's within what we're building to disambiguate - `RO` doesn't really have a lot of meaning outside this app.